tests/unittests/test_transform.py

```diff
@@ -0,0 +1,78 @@
+import unittest
+
+from parameterized import parameterized
+from tap_mixpanel.transform import transform_record
+
+class TestTransformRecord(unittest.TestCase):
+    """
+    Test case for transforming record.
+    """
+
+    test_dict_1 = {
+        '$distinct_id': '1234',
+        '$properties': {
+            '$key1': 'value1',
+            '$key2': 'value2'
+        }
+    }
```

Comment:
Replace `'` to `"` strings in this test file since in other tests we have used `"` strings.

Reply:
Done